xla/pjrt/gpu/se_gpu_pjrt_client_test.cc

```diff
@@ -800,24 +800,35 @@ absl::StatusOr<std::unique_ptr<PjRtBuffer>> CreateDeviceBufferForTest(
   return input;
 }
 
+constexpr char const* kD2HProgram = R"(
```

Comment:
maybe constexpr char* is enough? or constexpr char kD2HProgram[]?

Please note that this is compile-time literal so no need for the extra const. It might be better to align with the other location of the code base.

Reply:
> Please note that this is compile-time literal so no need for the extra const. It might be better to align with the other location of the code base.

It is unclear what to align with - I actually took the same incantation that is already used in this very file (see line 99 above).